reactor-netty-core/build.gradle

```diff
@@ -54,6 +54,7 @@ dependencies {
 	api "io.netty:netty-buffer:$nettyByteBufVersion"
 	api "io.netty:netty5-handler:$nettyVersion"
 	api "io.netty.contrib:netty-handler-proxy:$nettyContribVersion"
+	api "io.netty.contrib:netty-codec-multipart:$nettyContribVersion"
```

Comment:
reactor-netty-core must not depend on HTTP features

Reply:
thanks, this dependency was added by mistake, I have removed it.